anthos-bm-gcp-terraform/terraform.tfvars.sample

```diff
@@ -6,6 +6,7 @@ abm_cluster_id                = "cluster1"
 anthos_service_account_name   = "baremetal-gcr"
 resources_path                = "./resources"
 machine_type                  = "n1-standard-8"
+service_account.              = "<GCE service account with iam.serviceAccountKeyAdmin>"
```

Comment:
```suggestion
# service_account               = "<GCE service account with iam.serviceAccountKeyAdmin permission>"
```

Make is as a commented variable and move it below "instance_count". We don't want to force everyone to create this extra service account. Ideally we want a very smooth friction free quick test path